cypress/components/popover-container/popover-container.cy.tsx

```diff
@@ -380,19 +427,17 @@ context("Test for Popover Container component", () => {
       cy.get(`[role="checkbox"]`).eq(0).check();
       cy.get(`[role="checkbox"]`).eq(1).check();
       cy.get(`[role="checkbox"]`).eq(2).check();
-      getDataElementByValue("main-text")
-        .eq(1)
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
+      getDataElementByValue("main-text").eq(1).click();
```

Comment:
**suggestion(non-blocking)**: since we are rendering three buttons here, one corresponding to each filter option, what do you think about asserting the visibility of the other two buttons?

That way we can verify if the all three buttons are visible before the axe check begins.

```tsx
it("should pass accessibilty tests for Popover Container Filter story, with all options selected and apply button clicked", () => {
   // ...
  getDataElementByValue("main-text").eq(1).click();
  getComponent("pill").eq(0).should("be.visible");
  getComponent("pill").eq(1).should("be.visible");
  getComponent("pill").eq(2).should("be.visible");
  cy.checkAccessibility();
});
```